RadixWallet/Features/ImportMnemonic/VerifyMnemonic/VerifyMnemonic.swift

```diff
@@ -0,0 +1,93 @@
+import ComposableArchitecture
+import SwiftUI
+
+// MARK: - DisplayMnemonic
+public struct VerifyMnemonic: Sendable, FeatureReducer {
+	public struct State: Sendable, Hashable {
+		public static let numberOfWordsToConfirm = 3
```

Comment:
minor, but this ought to be `numberOfRandomWordsToConfirm` since it is in fact `3 + 1`. Maybe a comment?